### ISSUE TITLE
Not working within fragment when activity was created already

### ISSUE THREAD

**Issue by sjd753** [recruit-mp/LightCalendarView#30]
If you put calendarView inside fragment and put that fragment in an activity and start the activity, it works fine but if you have a drawer activity (say) and it already running and when a navigation drawer item is selected and you replace another fragment with this calendarFragment it does show up at all. It's a bug i think. Help required ASAP please!

**Comment by TKolbusz** [comment]
Yep, I have the same problem. I think layout height and width isn't calculated as it should be when instantiating view at runtime.

**Comment by sjd753** [comment]
I kinda agree with "I think layout height and width isn't calculated as it should be " but not with " when instantiating view at runtime" coz when you use calendatView inside a fragment that is attached to its activity in onCreate() then there's no problem! Weird!

**Comment by sjd753** [comment]
**Basically if the calendarView is instantiated in the onCreate() of an activity whether or not it be in a fragment (fragment attached in onCreate) or not , then there's no issue. If it's not instantiated in onCreate() then calendar does not appear.** _Although I've noticed that if the activity was restarted forcefully (say by the instant run) then calendar reappears!_
